apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -595,6 +595,18 @@ open class ChatController(var ctrl: ChatCtrl?, val ntfManager: NtfManager, val a
     }
   }
 
+  suspend fun apiGetChatItemInfo(type: ChatType, id: Long, itemId: Long): ChatItemInfo? {
+    return when (val r = sendCmd(CC.ApiGetChatItemInfo(type, id, itemId))) {
+      is CR.ApiChatItemInfo -> r.chatItemInfo
+      else -> {
+        if (!(networkErrorAlert(r))) {
```

Comment:
how can it be a network error in this api

Reply:
:)